generated/nidaqmx/nidaqmx.proto

```diff
@@ -3852,9 +5751,9 @@ enum DeviceInt32AttributeValues {
   DEVICE_INT32_ANT_STATUS_ABSENT = 15994;
   DEVICE_INT32_ANT_STATUS_OVERCURRENT = 15995;
   DEVICE_INT32_BUS_TYPE_PCI = 12582;
-  DEVICE_INT32_BUS_TYPE_PC_IE = 13612;
+  DEVICE_INT32_BUS_TYPE_PCIE = 13612;
   DEVICE_INT32_BUS_TYPE_PXI = 12583;
-  DEVICE_INT32_BUS_TYPE_PX_IE = 14706;
+  DEVICE_INT32_BUS_TYPE_PXIE = 14706;
```

Comment:
This breaks source compatibility but arguably it fixes a bug.

@astarche @reckenro

Reply:
Sure Brad. So, are we good with this change? or we should fix this as well?

Reply:
I think this change is okay. @astarche @reckenro do you agree?

Reply:
I think this change is okay as well. @maxxboehme , does this sound good to you?

If we do make this change can you add the `source-breaking` label to this PR?